src/Form/Field/LayoutField.php

```diff
@@ -61,13 +138,13 @@ public function attrsForm(): Form
 	{
 		return new Form(
 			fields: $this->settings()?->fields() ?? [],
-			model:  $this->model
+			model:  $this->model()
 		);
 	}
 
 	public function layouts(): array
 	{
-		return $this->layouts ?? [['1/1']];
+		return array_map(Str::split(...), $this->layouts ?? [['1/1']]);
```

Comment:
```suggestion
		return array_map(Str::split(...), $this->layouts ?? ['1/1']);
```

`[['1/1']]` would be the fallback value after running itself through array_map.